webapp/Makefile

```diff
@@ -4,6 +4,12 @@ include config.mk
 # The CI environment variable is set automatically in CircleCI and GitLab CI
 CI ?= false
 
+# Detect Linux/aarch64 and set a flag.
```

Comment:
Would you mind adding a bit more detail about why we're setting this flag? Something like this would work
```suggestion
# Detect Linux/aarch64 and set a flag to fix an issue with optipng-bin on it
```

Reply:
This had been updated.